src/platform_impl/macos/view.rs

```diff
@@ -484,7 +485,10 @@ declare_class!(
             }
             let was_in_preedit = self.state.ime_state == ImeState::Preedit;
 
-            let characters = get_characters(event, false);
+            let ev_mods = event_mods(event);
+            let option_as_alt = self.window().is_option_as_alt();
+            let ignore_alt_characters = option_as_alt && ev_mods.alt() && !ev_mods.ctrl();
```

Comment:
I think you'd want to ignore it if `ev_mods.logo()` is set as well - at least that's what I think Terminal.app does?